sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/transformable/sierra/SierraRecordNumbers.scala

```diff
@@ -0,0 +1,77 @@
+package uk.ac.wellcome.models.transformable.sierra
+
+object SierraRecordTypes extends Enumeration {
+  val bibs, items = Value
+}
+
+/** Utilities for working with Sierra "record numbers".
+  *
+  * These record numbers are the seven- or eight-digit IDs that
+  * are used to identify resources in Sierra.
+  */
+object SierraRecordNumbers {
```

Comment:
I’m about to add some more methods related to Sierra record numbers, and having them all in one place seemed sensible.